examples/egui/editor.rs

```diff
@@ -0,0 +1,303 @@
+use crate::fox::{ Fox, FoxAnimClip };
+use dotrix::{
+    animation::State as AnimState,
+    components::{ Animator, Light, Model },
+    ecs::{ Const, Mut },
+    egui::{
+        CollapsingHeader,
+        combo_box,
+        DragValue,
+        Egui,
+        Grid,
+        SidePanel,
+        Slider,
+    },
+    input::{ Button, State as InputState },
+    math::{ Point3, Vec3, Vec4 },
+    renderer::transform::Transform,
+    services::{ Camera, Frame, Input, Renderer, World },
+};
+use std::f32::consts::PI;
+
+pub struct Editor {
```

Comment:
It is not an editor really... Maybe consider other name? Sidebar, Controls, Settings, Options...

Reply:
maybe `Settings`?

Reply:
yes, can be